modules/elasticsearch/charts.go

```diff
@@ -0,0 +1,581 @@
+			Units: "operations/s",
+			Fam:   "indices flush",
+			Ctx:   "elasticsearch.node_indices_flush",
+			Dims: Dims{
+				{ID: "node_indices_flush_total", Name: "flush", Algo: module.Incremental},
+			},
+		},
+		{
+			ID:    "node_indices_flush_operations_time",
+			Title: "Time Spent On Flush Operations",
+			Units: "milliseconds",
+			Fam:   "indices flush",
+			Ctx:   "elasticsearch.node_indices_flush_time",
+			Dims: Dims{
+				{ID: "node_indices_flush_total_time_in_millis", Name: "flush", Algo: module.Incremental},
+			},
+		},
+		// Indices Fielddata
+		{
+			ID:    "node_indices_fielddata_memory_usage",
+			Title: "Fielddata Cache Memory Usage",
+			Units: "bytes",
+			Fam:   "indices fielddata",
+			Ctx:   "elasticsearch.node_indices_fielddata_memory_usage",
+			Type:  module.Area,
+			Dims: Dims{
+				{ID: "node_indices_fielddata_memory_size_in_bytes", Name: "used"},
+			},
+		},
+		{
+			ID:    "node_indices_fielddata_evictions",
+			Title: "Fielddata Evictions",
+			Units: "operations/s",
+			Fam:   "indices fielddata",
+			Ctx:   "elasticsearch.node_indices_fielddata_evictions",
+			Dims: Dims{
+				{ID: "node_indices_fielddata_evictions", Name: "evictions", Algo: module.Incremental},
+			},
+		},
+		// Indices Segments
+		{
+			ID:    "node_indices_segments_count",
+			Title: "Segments Count",
+			Units: "segments",
+			Fam:   "indices segments",
+			Ctx:   "elasticsearch.node_indices_segments_count",
+			Dims: Dims{
+				{ID: "node_indices_segments_count", Name: "segments"},
+			},
+		},
+		{
+			ID:    "node_indices_segments_memory_usage_total",
+			Title: "Segments Memory Usage Total",
+			Units: "bytes",
+			Fam:   "indices segments",
+			Ctx:   "elasticsearch.node_indices_segments_memory_usage_total",
+			Dims: Dims{
+				{ID: "node_indices_segments_memory_in_bytes", Name: "used"},
+			},
+		},
+		{
+			ID:    "node_indices_segments_memory_usage",
+			Title: "Segments Memory Usage",
+			Units: "bytes",
+			Fam:   "indices segments",
+			Ctx:   "elasticsearch.node_indices_segments_memory_usage",
+			Type:  module.Stacked,
+			Dims: Dims{
+				{ID: "node_indices_segments_terms_memory_in_bytes", Name: "terms"},
+				{ID: "node_indices_segments_stored_fields_memory_in_bytes", Name: "stored_fields"},
+				{ID: "node_indices_segments_term_vectors_memory_in_bytes", Name: "term_vectors"},
+				{ID: "node_indices_segments_norms_memory_in_bytes", Name: "norms"},
+				{ID: "node_indices_segments_points_memory_in_bytes", Name: "points"},
+				{ID: "node_indices_segments_doc_values_memory_in_bytes", Name: "doc_values"},
+				{ID: "node_indices_segments_index_writer_memory_in_bytes", Name: "index_writer"},
+				{ID: "node_indices_segments_version_map_memory_in_bytes", Name: "version_map"},
+				{ID: "node_indices_segments_fixed_bit_set_memory_in_bytes", Name: "fixed_bit_set"},
+			},
+		},
+		// Indices Translog
+		{
+			ID:    "node_indices_translog_operations",
+			Title: "Translog Operations",
+			Units: "operations",
```

Comment:
And here?

Reply:
i believe this one is not `incremental` too